QuizApp/LoginView/LoginViewController.swift

```diff
@@ -62,42 +105,66 @@ class LoginViewController: UIViewController {
                 make.centerX.equalToSuperview()
             }
         }
-
-        titleLabel.isHidden = true
-
+        
+        titleLabel.snp.makeConstraints { make in
+            make.top.equalTo(view).offset(height * 0.1)
+        }
+        
         emailField.snp.makeConstraints { make in
             make.top.equalTo(view).offset(height * 0.2)
         }
-
+        
         passwordField.snp.makeConstraints { make in
             make.top.equalTo(emailField).offset(45 + 18)
         }
-
+        
         errorLable.snp.makeConstraints { make in
             make.top.equalTo(passwordField).offset(38)
         }
-
+        
         loginButton.snp.makeConstraints { make in
             make.top.equalTo(errorLable).offset(38)
         }
         loginButton.layer.cornerRadius = 20
-
+        
     }
-
+    
 }
 
 extension LoginViewController: LoginPresenterDelegate {
     func loginSuccess() {
         loginButton.isEnabled = true
         print("Sucessful login")
         errorLable.isHidden = true
-        presenter.showQuizzes()
+        animatedRemove()
+        DispatchQueue.main.asyncAfter(deadline: .now()+2.0) {
+            self.presenter.showQuizzes()
+        }
```

Comment:
It would be better if showQuizzes() was called in a completion block of the last animated UI element.